addons/website_sale_stock/static/src/js/variant_mixin.js

```diff
@@ -82,6 +82,15 @@ VariantMixin._onChangeCombinationStock = function (ev, $parent, combination) {
 };
 
 publicWidget.registry.WebsiteSale.include({
+
+    events: Object.assign(publicWidget.registry.WebsiteSale.prototype.events, {
+         'click .show_notification_email_form': '_onClickShowNotificationEmailForm'
+    }),
```

Comment:
This is modifying in place and assigning events at the same time
```suggestion
    events: Object.assign({}, publicWidget.registry.WebsiteSale.prototype.events, {
         'click .show_notification_email_form': '_onClickShowNotificationEmailForm'
    }),
```